src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -128,6 +140,107 @@ public async Task<IActionResult> Create(CreateApiViewModel model)
             return BadRequest(ModelState);
         }
 
+        [HttpPost]
+        [Route("disable/{tenantName}")]
+        public async Task<IActionResult> Disable(string tenantName)
+        {
+            if (!_currentShellSettings.IsDefaultShell())
+            {
+                return Forbid();
+            }
+
+            if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageTenants))
+            {
+                return this.ChallengeOrForbid("Api");
+            }
+
+            if (!_shellHost.TryGetSettings(tenantName, out var shellSettings))
+            {
+                return NotFound();
+            }
+
+            if (shellSettings.State != TenantState.Running)
+            {
+                return BadRequest(S["You can only disable an Enabled tenant."]);
```

Comment:
```suggestion
                return BadRequest(S["You can only disable a Running tenant."]);
```

Reply:
I'm using the same message used in the `AdminController`, do you want me to modify it in both place?

Reply:
Yes, I'd use the same.

Reply:
Okay done in both places.